composer.json

```diff
@@ -22,7 +22,7 @@
         "doctrine/inflector": "^2.0",
         "dragonmantank/cron-expression": "^3.1",
         "egulias/email-validator": "^3.1",
-        "league/commonmark": "^1.3|^2.0",
```

Comment:
the other composer.json files need to be updated too

Reply:
Thanks, forgot about that.